docs/UserGuide.md

```diff
@@ -316,20 +405,27 @@ Examples:
 * `sort a`
 * `sort s`
 * `sort e`
+
 ![sort example](images/sort.png)
 
+:information_source: Remarks:
+- After sorting by start date or end date, you noticed that application that does not have their dates displayed is in 
+between applications with their dates displayed, please refer to our [FAQ](#faq-questionquestionquestion).
+- After adding an application or editing the dates of an application, the sort command has to be run again,
```

Comment:
i think not necessarily dates, since can edit the name as well and can sort by name. Also i think we need to standardize the adding of an application OR adding of a company, because you can add the company in but havent applied yet. and at the top in the addcommand and those commands at the top it was about adding companies.

maybe can state that when adding a new company it will be added to the bottom of the list, and as for editing the arrangement will stay the same